saleor/payment/gateways/stripe/tests/test_plugin_deprecated.py

```diff
@@ -0,0 +1,84 @@
+from unittest.mock import Mock, patch
```

Comment:
Use relative imports here instead of absolute. (we use relative almost in all places)